TezosChain.ts

```diff
@@ -734,62 +734,67 @@ export class TezosChain extends pulumi.ComponentResource {
         },
         { provider: this.provider }
       )
-      if (name.includes("dailynet")) {
 
-        // Define a custom awaiter function
-        const waitForDalIps = async () => {
-          while (true) {
-            // Get the status of the service from the live object
-            const dalBootstrapLbStatus = await dalBootstrapLb.status;
-            const dal1LbStatus = await dal1Lb.status;
-
-            // Check if the service is ready and the IP address is available
-            if (dalBootstrapLbStatus && dalBootstrapLbStatus.loadBalancer && dalBootstrapLbStatus.loadBalancer.ingress[0].ip &&
-              dal1LbStatus && dal1LbStatus.loadBalancer && dal1LbStatus.loadBalancer.ingress[0].ip) {
-              break;
-            }
-
-            // Wait for 10 seconds before the next check
-            await new Promise((resolve) => setTimeout(resolve, 10000));
+      params.helmValues.node_config_network.dal_config.bootstrap_peers = [
+        `${dalBootstrapP2pFqdn}:11732`,
+      ]
+
+      // Wait for DAL LB to get their IP addresses.
+      let dal1LbIp;
+      let dalBootstrapLbIp;
+      const waitForDalIps = async () => {
+        while (true) {
+          // Get the status of the service from the live object
+
+          dal1LbIp = (k8s.core.v1.Service.get(name, `${name}-dal-dal1`).status)?.loadBalancer?.ingress?.[0]?.ip;
+          dalBootstrapLbIp = (k8s.core.v1.Service.get(name, `${name}-dal-bootrstrap`).status)?.loadBalancer?.ingress?.[0]?.ip;
```

Comment:
both these lines are async and i think need to be awaited. Otherwise the variables are just being assigned promises